src/CLI/clitree/cli-xml/acl.xml

```diff
@@ -0,0 +1,218 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
As discussed in our workgroup meetings there was no plan to auto generate CLI XML, actioner and renderer python scripts as we had to support IS CLI (Cisco like CLI)
Also since we have taken openConfig yang in this case, in all practicality we could not do that.